python/cudf/cudf/core/column/decimal.py

```diff
@@ -204,22 +200,17 @@ def normalize_binop_value(self, other) -> Self | cudf.Scalar:
                     other = other.astype(self.dtype)
             return other
         if isinstance(other, cudf.Scalar) and isinstance(
-            # TODO: Should it be possible to cast scalars of other numerical
-            # types to decimal?
             other.dtype,
-            cudf.core.dtypes.DecimalDtype,
+            DecimalDtype,
         ):
+            # TODO: Should it be possible to cast scalars of other numerical
+            # types to decimal?
```

Comment:
I think this is ultimately the same question as https://github.com/rapidsai/cudf/issues/7680 CC @brandon-b-miller